USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -217,7 +213,7 @@ private void RenderFootnotes()
 
                     foreach(Marker marker in footnoteKVP.Value.Contents)
                     {
-                        RenderMarker(marker, renderFootnote);
+                        RenderMarker(marker, new StyleConfig(),renderFootnote);
```

Comment:
Why are you adding a new style config here. It should just be using its parents.

Reply:
Only top level rendering should be creating a new StyleConfig because creating a new one breaks any style being handed down from above.

Reply:
I created a StyleConfig param in the RenderFootnotes method, so that the cMarker will continue to pass the parent styleconfig down to the footnotes.